include/VecCore/Backend/Implementation.h

```diff
@@ -275,6 +275,7 @@ T Blend(const Mask<T> &mask, const T &src1, const T &src2)
 
 // Miscellaneous
 
+// Note that SIMDsizeUpTo() is more general, as the code below is equivalent to SIMDsizeUpTo<VeryLargeInt>()
```

Comment:
humm ... For the general case, it is usually 'bad' to have the doc for a routine separated from his definition and/or implementation.  This usually means that the two will easily go out-of-sync.  Why not use doxygen to generate the reference guide?